src/wallet/wallet.cpp

```diff
@@ -1970,6 +1970,32 @@ bool CWallet::SelectCoins(const CAmount& nTargetValue, set<pair<const CWalletTx*
     return res;
 }
 
+#if 0
```

Comment:
why adding a function tat is never used and then disable it via `#if 0`

Reply:
it was part of the merge.  It seems like a useful function but I haven't refactored to use it yet.